torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -1,4 +1,5 @@
 #include <torch/csrc/jit/runtime/profiling_graph_executor_impl.h>
+#include <torch/csrc/jit/codegen/cuda/parser.h>
```

Comment:
parser can't be exposed here.
I'll work it out in our API to add a wrapper for it.